route53_crossfeed_app.tf

```diff
@@ -243,6 +243,16 @@ resource "aws_route53_record" "crossfeed_staging_CNAME2" {
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
```

Comment:
I can't apply this `CNAME` record because there exist other non-`CNAME` records for `staging.crossfeed.cyber.dhs.gov`.

Reply:
@jsf9k  What can we do for this issue? I can replicate these changes with any alterations needed in a new PR from one of my own forks if needed.  Not sure what exactly needs to change.

Reply:
I'm not sure what needs to be done.  We can just wait for @epicfaace to return on Monday.

Reply:
Can you explain more why this is an issue? Why would non-CNAME records on the same domain prevent adding a CNAME record to it?

Reply:
[It's a fundamental constraint due to the way `CNAME` records are handled in DNS](https://serverfault.com/questions/853481/cant-add-dns-cname-record#:~:text=A%20CNAME%20is%20just%20an,touch%20your%20DNS%20for%20this!).

Reply:
@jsf9k the records you are referring to then are the MX and TXT records that are also listed?  Is it just a matter of moving all of the records to the new domain?

Reply:
@stewartl97 - It depends how you choose to fix it.  You can get rid of the `CNAME` record and explicitly list all records, or you can keep the `CNAME` and delete the other records.  I don't know what is best for your use case.